sdk/postgresql/Azure.ResourceManager.PostgreSql/CHANGELOG.md

```diff
@@ -1,22 +1,37 @@
 # Release History
 
-## 1.2.0-beta.7 (Unreleased)
+## 1.2.0 (2024-11-05)
```

Comment:
is this referring to release date?

Reply:
Yes it needed to be a future date, I updated to November Ship Date.